### PR TITLE
update: webpackTask config (KT-42421)

### PR DESCRIPTION
**Pull request by p7nov**
[KT-42421](https://youtrack.jetbrains.com/issue/KT-42421)
Add config parameters available in webpackTask:
- `outputFileName`
- `output.libraryTarget`

### CODE REVIEW

**Review by SebastianAigner — APPROVED** [review]
LGTM – I have added one commit to change `commonjs` to `commonjs2` in the example snippet. Explanation for this from https://kotlinlang.org/docs/reference/js-modules.html:
> in most cases, you probably want commonjs2, which adds the module.exports syntax to the generated library [...]

**Review by ilgonmic — APPROVED** [review]

File: pages/docs/reference/js-project-setup.md [hunk 1]
```diff
@@ -365,7 +365,21 @@ The Kotlin/JS Gradle plugin automatically generates a standard webpack configura
 at build time which you can find the at `build/js/packages/projectName/webpack.config.js`.
 
 The most common webpack adjustments can be made directly via the
-`kotlin.js.browser.webpackTask` configuration block in the Gradle build file.
+`kotlin.js.browser.webpackTask` configuration block in the Gradle build file. These are:
+- `outputFileName` - the name of the webpacked output file. It will be generated in `<projectDir>/build/distibution/` after
+an execution of a webpack task. The default value is the project name.
+- `output.libraryTarget` - the module system for the webpacked output. Learn more about available module systems for
+Kotlin/JS projects [here](js-modules.html). The default value is `umd`.
```
Comment (katia-energizer) [inline pages/docs/reference/js-project-setup.md:372]: Please avoid using here as linked text. It doesn't add any value. Make real text `available module systems for Kotlin/JS` a link.

File: pages/docs/reference/js-project-setup.md [hunk 2]
```diff
@@ -365,7 +365,21 @@ The Kotlin/JS Gradle plugin automatically generates a standard webpack configura
 at build time which you can find the at `build/js/packages/projectName/webpack.config.js`.
 
 The most common webpack adjustments can be made directly via the
-`kotlin.js.browser.webpackTask` configuration block in the Gradle build file.
+`kotlin.js.browser.webpackTask` configuration block in the Gradle build file. These are:
```
Comment (katia-energizer) [inline pages/docs/reference/js-project-setup.md:368]: When I first read the sentence, I didn't get what `These are` relate to. After reading twice, I understood. But I am not sure you need this phrase and it adds value. You can just put the semicolon after the first sentence:
> The most common webpack adjustments can be made directly via the
`kotlin.js.browser.webpackTask` configuration block in the Gradle build file:

File: pages/docs/reference/js-project-setup.md [hunk 3]
```diff
@@ -365,7 +365,21 @@ The Kotlin/JS Gradle plugin automatically generates a standard webpack configura
...
+- `outputFileName` - the name of the webpacked output file. It will be generated in `<projectDir>/build/distibution/` after
+an execution of a webpack task. The default value is the project name.
+- `output.libraryTarget` - the module system for the webpacked output. Learn more about available module systems for
+Kotlin/JS projects [here](js-modules.html). The default value is `umd`.
+
+<div class="sample" markdown="1" mode="groovy" theme="idea">
```
Comment (katia-energizer) [inline pages/docs/reference/js-project-setup.md:374]: Is the code the same for Kotlin and Groovy?

**Review by katia-energizer — APPROVED** [review]
Everything is OK. A couple of comments to improve readability.

File: pages/docs/reference/js-project-setup.md (hunk 3 above)
Comment (p7nov) [reply pages/docs/reference/js-project-setup.md:374]: Yes, no difference here.

**Pull request merged**